### PR TITLE
fixes: #3956 Cleaned up knapsack and images directory

### PR DESCRIPTION
**Pull request by dhruvmanila**
Fixes: #3956 

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by dhruvmanila** [comment]
There's this [README](https://github.com/TheAlgorithms/Python/blob/master/knapsack/README.md) file as well which probably either needs updating or should we just remove it?

**Comment by cclauss** [comment]
Perhaps we could get the original poster @tbsschroeder to update the README to reflect that all implementations of the knapsack problem are going to be put in one directory.

Let's do this.  Move the README along with the code and we will see if someone is willing to update it appropriately.  If it is not updated by yearend, we can whack the README.

**Comment by dhruvmanila** [comment]
Well, then this is good to go. I haven't touched the README so it is present in the `knapsack` directory.

File: knapsack/tests/test_greedy_knapsack.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from . import greedy_knapsack as kp
+from knapsack import greedy_knapsack as kp
```
Comment (cclauss) [inline knapsack/tests/test_greedy_knapsack.py:3]: Is this a vital change?  There are two ways to go.  Hardcode the name of the parent directory or use relative imports.  Each has its pros and cons.  Relative imports are better for easy refactoring and absolute imports are better for libraries and projects that need stability.

**Review by cclauss — APPROVED** [review]

File: knapsack/tests/test_greedy_knapsack.py (hunk 1 above)
Comment (dhruvmanila) [reply knapsack/tests/test_greedy_knapsack.py:3]: Relative import would be like so: `from .. import greedy_knapsack as kp` as I created a `tests` directory for both test files. Here's the tree:
```
knapsack/
├── README.md
├── __init__.py
├── greedy_knapsack.py
├── knapsack.py
└── tests
    ├── __init__.py
    ├── test_greedy_knapsack.py
    └── test_knapsack.py
```

Comment (dhruvmanila) [reply knapsack/tests/test_greedy_knapsack.py:3]: I suppose for our use case relative imports should be de facto as we don't know from where the script is going to be called by the user.

**Comment by cclauss** [comment]
Nice work.  Thanks for doing this.

**Pull request merged**